data/json/items/book/misc.json

```diff
@@ -931,13 +931,105 @@
+    "snippet_category": [
+      {
+        "id": "philosophy1",
+        "text": "This is a copy of Nietzsche's \"Beyond Good and Evil\". Its cover is dog-eared and creased."
+      },
+      {
+        "id": "philosophy2",
+        "text": "This is a copy of \"The Unique and Its Property\" by Max Stirner. A modern translation by Wolfi Landstreicher."
+      },
+      {
+        "id": "philosophy3",
+        "text": "This is a copy of Jean-Paul Sartre's \"Being and Nothingness\". A key work in the existentialist tradition."
+      },
+      {
+        "id": "philosophy4",
+        "text": "A large, extended version of \"Madness and Civilisation\" by Michel Foucault. The cover features a striking image of a Panopticonic Prison."
+      },
+      {
+        "id": "philosophy5",
+        "text": "This is a copy of \"The Postmodern Condition: A Report on Knowledge\" by Lyotard."
+      },
+      {
+        "id": "philosophy6",
+        "text": "A collection of texts and essays by Jacques Derrida. It's pages loose and yellowed, you should probably handle it with care."
+      },
+      {
+        "id": "philosophy7",
+        "text": "This is a copy of \"Society of the Spectacle\" by Guy Debord. Its cover shows rows of adults staring placidly into a screen."
+      },
+      {
+        "id": "philosophy8",
+        "text": "This is a split copy of both \"An Ethic of Sexual Difference\" and \"This Sex Which Is Not One\" by Luce Irigaray."
+      },
+      {
+        "id": "philosophy9",
+        "text": "This is a copy of Baudrillard's \"Simulation and Simulacra\". The cover contains an image of a man holding a colored pill in each hand, with the caption \"Welcome to the Desert of the Real.\". You think you've seen this movie."
+      },
+      {
+        "id": "philosophy10",
+        "text": "This is a small, pocket version of Sartre's \"Existentialism and Humanism.\" It looks to have been used as a coaster in a past life."
+      },
+      {
+        "id": "philosophy11",
+        "text": "This is a copy of \"Practical Ethics\" by Peter Singer. From the local university press."
+      },
+      {
+        "id": "philosophy12",
+        "text": "This is a photocopied spiral-bound copy of \"Industrial Society and Its Future\" by 'Freedom Club'. The original looks to have been written on a typewriter before being copied."
+      },
+      {
+        "id": "philosophy13",
+        "text": "This is a copy of \"Industrial Society and Its Future\" by Ted Kaczynski. Its cover is an image of a hand-crafted wooden box filled with wiring and an ominous looking metal tube. Provocative."
+      },
+      { "id": "philosophy14", "text": "This is a small reader on Hegel's Dialectics." },
+      {
+        "id": "philosophy15",
+        "text": "This is a copy of \"The State and Revolution\" by Vladimir Lenin. In English, thankfully."
+      },
+      { "id": "philosophy16", "text": "This is a copy of \"In Defence of Terrorism\" by Leon Trotsky." },
```

Comment:
In Defence of... Terrorism? 👀

Reply:
Yes.  This is a book.  I'm finding it hard to see your objection as genuine here because a quick search online will show you that this exists.

https://www.amazon.com/Defence-Terrorism-Communism-Kautsky-Classic/dp/1330379012

Reply:
That is the title of an English edition, so I didn't find it because I was searching from a marxist archive site of some sort... Anyway, the exact title was "*The* Denfence of Terrorism", so it still needed to be corrected.